src/app/features/register-student/register-student.component.html

```diff
@@ -0,0 +1,83 @@
+<div class="container">
+    <div class="register-form-students">
+        <form class="form-students" [formGroup]="formGroup">
+            <div class="example-container">
+                <h1 class="title"> Registro de Estudiantes</h1>
+                <mat-form-field class="seccion-register"appearance="fill">
+                    <mat-label>Identificación Estudiante</mat-label>
+                    <input matInput name="idStudent" formControlName="idStudent" type="number"
+                    placeholder="Identificación"  required>
```

Comment:
```suggestion
                    <input matInput
                        name="idStudent"
                        formControlName="idStudent"
                        type="number"
                        placeholder="Identificación"
                        required>
```

Reply:
Required should not be needed here since you are adding in a reactive way, check other inputs too